misk-schema-migrator-gradle-plugin/src/main/kotlin/misk/schemamigratorgradleplugin/MiskSchemaMigratorExtension.kt

```diff
@@ -0,0 +1,25 @@
+package misk.schemamigratorgradleplugin
```

Comment:
Still mulling on what would be better to do here but the full name is killing me in length and unreadability.

Maybe `misk.gradle.schemamigrator`?